parts/kuberneteskubelet.service

```diff
@@ -45,6 +45,7 @@ ExecStart=/usr/bin/docker run \
         --azure-container-registry-config=/etc/kubernetes/azure.json \
         --hairpin-mode=promiscuous-bridge \
         --network-plugin=${KUBELET_NETWORK_PLUGIN} \
+        --feature-gates=Accelerators=true \
```

Comment:
This feature is only on 1.6.x. We need to be able to not set it for 1.5.x clusters.